### DISCUSSION TITLE
Android version?

### DISCUSSION THREAD

**Discussion by leo150** [VLprojects/mediasoup-client-swift#10] [General]
Hi, thanks a lot for your work on this project. Do you plan to open-source Android version?

**Comment by fedulvtubudul** [comment]
We use this build scripts for mediasoup client on Android platform: https://github.com/foxaice/mediasoup-android-client-builder. It's not updated for latest versions WebRTC and Mediasoup yet. I hope we will update it some day, but not so fast, I afraid.
